src/en/news/blog/2025/rbd-live-migration/index.md

```diff
@@ -0,0 +1,478 @@
+[ceph: root@rbd-client /]# rbd migration commit c2pool1/c2image1 --cluster c2
+Commit image migration: 100% complete...done.
+```
+
+### Supported Image Formats
+
+Live migration supports three primary formats:
+
+1. *Native format* uses Ceph's internal format for efficient migration.
+
+The native format does not include the stream since it utilizes
+native Ceph operations. For example, to import from the
+image ``rbd/ns1/image1@snap1``, the source specification can be constructed as below:
+
+```
+{
+"type": "native",
+"pool_name": "rbd",
+"pool_namespace": "ns1",
+"image_name": "image1",
+"snap_name": "snap1"
+}
+```
+
+2. *QCOW format* is compatible with QEMU Copy-On-Write volume images.
+
+The QCOW format describes a QEMU copy-on-write (QCOW) block device.
+QCOW v1 and v2 formats are currently supported, with the exception of
+certain features including compression, encryption, backing files, and external
+data files. Use the QCOW format with any supported stream source:
+
+```
+{
+    "type": "qcow",
+    "stream": {
+      "type": "file",
+  "file_path": "/mnt/image.qcow"
+    }
+}
+```
+
+3. *Raw format* is used for thick-provisioned block device exports.
+
+```
+{
+    "type": "raw",
+    "stream": {
+      "type": "file",
+      "file_path": "/mnt/image-head.raw"
+    },
+    "snapshots": [
+        {
+            "type": "raw",
+            "name": "snap1",
+            "stream": {
+              "type": "file",
+       "file_path": "/mnt/image-snap1.raw"
+            }
+        },
+    (optional oldest to newest ordering of snapshots)
+}
+```
+
+### Supported Streams
+
+Multiple stream types are available for importing from various data sources:
+
+#### File Stream (Local files)
+
+Use a ``file`` stream to import from a locally accessible POSIX file source.
+
+```
+{
+    <format-specific parameters>
+    "stream": {
+        "type": "file",
+        "file_path": "FILE_PATH"
+    }
+}
+```
+#### HTTP Stream (remote web server)
+
+Use an ``HTTP`` stream to import from an HTTP or HTTPS web server.
+
+```
+{
+    <format-specific parameters>
+    "stream": {
+        "type": "http",
+        "url": "URL_PATH"
+    }
+}
+```
+
+#### S3 Stream (AWS or compatible object storage)
+
+Use an ``s3`` stream to import from an S3 bucket.
+
+```
+{
+    <format-specific parameters>
+    "stream": {
+        "type": "s3",
+        "url": "URL_PATH",
+        "access_key": "ACCESS_KEY",
+        "secret_key": "SECRET_KEY"
+    }
+}
+```
+
+#### NBD Stream (Network Block Device exports)
+
+Use an ``NBD`` stream to import from a remote NBD export.
+
+```
+{
+    <format-specific parameters>
+    "stream": {
+        "type": "nbd",
+        "uri": "<nbd-uri>",
+    }
+}
+```
+
+The ``nbd-uri`` parameter must follow the [NBD URI specification](https://github.com/NetworkBlockDevice/nbd/blob/master/doc/uri.md).  The default NBD port is ``tcp/10809``.
+
+### Use Cases of RBD Live Migration
+
+* Disaster Recovery and Data Migration\
```

Comment:
I can only assume that these backslashes at the end of these lines are an artifact of moving from one markup scheme to another. Anyway, I reckon they've got to go.